src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.tsx

```diff
@@ -10,7 +11,7 @@ const FavoritesCitiesWeatherWidget = () => {
       <CitiesWeatherWidget.Title>favorites</CitiesWeatherWidget.Title>
       <CitiesWeatherWidget.ShortcutsWrapper>
         {favouriteCities.map((city) => (
```

Comment:
**suggestion** I think we accepted the US spelling of `favourites` which is `favorites`. Consider changing it to US spelling to make it the same as anywhere else :)